sys/fs/cuse/cuse.c

```diff
@@ -1133,7 +1149,18 @@ cuse_server_ioctl(struct cdev *dev, unsigned long cmd,
 
 		pccmd->entered = curthread;
 
-		*pcmd = pccmd->sub;
+#ifdef COMPAT_FREEBSD64
+		if (!SV_CURPROC_FLAG(SV_CHERI)) {
+			pcmd64 = (struct cuse_command64 *)data;
+			pcmd64->dev = (uint64_t)pccmd->sub.dev;
+			CP(pccmd->sub, *pcmd64, fflags);
+			CP(pccmd->sub, *pcmd64, per_file_handle);
+			CP(pccmd->sub, *pcmd64, data_pointer);
```

Comment:
`per_file_handle` and `data_pointer` are capabilities as well, presumably these assignments also want casts with `__cheri_addr`?

Reply:
uintcap_t doesn't need such annotations, it's always address rather than offset